### PR TITLE
Add a section about data blobs.

### PR DESCRIPTION
**Pull request by afck**
In version 0.12 we added data blobs. This is a short section that explains how to use them, illustrated by `non-fungible`.

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
     - [Calling other Applications](developers/sdk/composition.md)
     - [Printing Logs from an Application](developers/sdk/logging.md)
     - [Writing Tests](developers/sdk/testing.md)
+    - [Using Data Blobs](developers/sdk/blobs.md)
```
Comment (christos-h) [inline src/SUMMARY.md:32]: Perhaps we put it above tests?

**Review by christos-h — APPROVED** [review]
Very nice!

File: src/SUMMARY.md (hunk 1 above)
Comment (afck) [reply src/SUMMARY.md]: I moved it up, also above the logs section.

File: src/developers/sdk/blobs.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Some applications may want to use static assets, like images or other data: e.g.
+the `non-fungible` example application implements NFTs, and each NFT has an
+associated image.
+
+Data blobs are pieces of binary data that, once published on _any_ chain, can be
+used on _all_ chains. What format they are in and what they are used for is
+determined by the application(s) that read(s) them.
+
+You can use the `linera publish-data-blob` command to publish the contents of a
```
Comment (ma2bd) [inline src/developers/sdk/blobs.md:11]: Do we have a graphql mutation as well?

**Review by ma2bd — APPROVED** [review]

File: src/developers/sdk/blobs.md (hunk 2 above)
Comment (afck) [reply src/developers/sdk/blobs.md:11]: Yes! The `non-fungible` README actually refers to that. I'll mention that one, too…

**Pull request merged**